python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/scenarios/auto_materialize_policy_scenarios.py

```diff
@@ -236,6 +236,7 @@
                 )
             ],
             expected_evaluations=[
+                AssetEvaluationSpec.empty("daily"),
```

Comment:
all of these changes are due to the fact that we're now writing all asset evaluations (empty or otherwise) in cases where the new evaluation is different from the previous one.

the end result is that in most cases, we'll get an empty evaluation written the tick after an asset is requested, as there will no longer be a reason to materialize